Source/WebCore/page/Page.cpp

```diff
@@ -1904,6 +1908,8 @@ void Page::finalizeRenderingUpdate(OptionSet<FinalizeRenderingUpdateFlags> flags
     }
 #endif
 
+    m_renderingUpdateRemainingSteps.last().remove(RenderingUpdateStep::CaretAnimation);
```

Comment:
Why do we need this here?

Reply:
because every RenderingUpdateStep must be removed at some point (Page has an assertion that checks this)

Reply:
But `runProcessingStep`does the `remove` for you. Not sure why you need it here. If you're asserting, something else is happening.